webrtc.html

```diff
@@ -7752,10 +7752,17 @@ <h3><dfn>RTCRtpTransceiver</dfn> Interface</h3>
       <a><code>RTCRtpReceiver</code></a> that share a common
       <code>mid</code>. As defined in <span data-jsep="rtptransceivers">[[!JSEP]]</span>,
       an <a><code>RTCRtpTransceiver</code></a> is said to be <dfn>associated</dfn> with
-      a <a>media description</a> if its <code><a data-link-for=RTCRtpTransceiver>mid</a></code>
-      property is non-null; otherwise it is said to be disassociated. Conceptually, an
-      <a>associated</a> transceiver is one that's represented in the last applied session
-      description.</p>
+      a <a>media description</a> if its "mid property" is non-null; otherwise it is
+      said to be <dfn>disassociated</dfn>.
+      Conceptually, a transceiver becomes <a>associated</a> once it has been
+      successfully represented in a session description. Once associated, a
+      transceiver only becomes <a>disassociated</a> if it is immediately rolled
+      back, or if the transceiver has been <a>stopped</a> and its m-line has
+      been recycled, as defined in <span data-jsep=
```

Comment:
I think the "if the transceiver has been stopped and its m-line has been recycled" should be part of the SLD/SRD steps.

Reply:
Why remove the link from "mid"?  Also, "successfully represented in a session description" is not as specific as "last applied session description".